parcels/particle.py

```diff
@@ -244,17 +246,25 @@ def execute(self, pyfunc=AdvectionRK4, time=None, dt=1., timesteps=1,
             current += output_steps * dt
             if output_file:
                 output_file.write(self, current)
+            if show_movie:
+                self.show(field=show_movie, t=current)
 
     def show(self, **kwargs):
-        import matplotlib.pyplot as plt
-        field = kwargs.get('field', None)
+        field = kwargs.get('field', True)
         lon = [p.lon for p in self]
         lat = [p.lat for p in self]
+        plt.ion()
+        plt.clf()
         plt.plot(np.transpose(lon), np.transpose(lat), 'ko')
-        if field is None:
+        if field is True:
+            axes = plt.gca()
+            axes.set_xlim([self.grid.U.lon[0], self.grid.U.lon[-1]])
+            axes.set_ylim([self.grid.U.lat[0], self.grid.U.lat[-1]])
             plt.show()
         else:
+            field = getattr(self.grid, field)
```

Comment:
I find passing the field itself, rather than it's name more intuitive and pythonic :snake:, so I would suggest something like:
```
if not isinstance(field, Field):
    field = getattr(self.grid, field)
```